README.md

```
@@ -1,310 +1,345 @@
-
-[Architecture](#architecture)
-
-[Vision](#place-2)
-
-[here](#place-2)
-
-## Installation
+### System Diagram
+Impact Graph acts as a central API gateway that connects the frontend application with various backend services and data sources:
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)